helpers/permissionsHandler.js

```diff
@@ -43,9 +43,39 @@ export const requestBluetoothPermissions = async () => {
 };
 
 export const requestNotificationPermissions = async () => {
+    await requestNotifications(['alert', 'sound']).then(({ status, settings }) => {
+      console.log("Notification Permission Status: " + status);
+    });
+};
+
+export const requestAllPermissions = async () => {
+  if (Platform.OS === OS.ANDROID) {
+    const currentPermissions = await checkMultiple([
+      PERMISSIONS.ANDROID.POST_NOTIFICATIONS
+    ])
+    const hasNotificationPermission = currentPermissions["android.permission.POST_NOTIFICATIONS"] === 'granted'
+
+    if (!hasNotificationPermission) {
+      await requestNotifications(['alert', 'sound']).then(({ status, settings }) => {
+        console.log("Notification Permission Status: " + status);
+        requestLocationPermissions();
+      });
+    } else {
+      await requestLocationPermissions();
+    }
+  }
+
   if (Platform.OS === OS.IOS) {
+    await requestBluetoothPermissions();
     await requestNotifications(['alert', 'sound']).then(({ status, settings }) => {
       console.log("Notification Permission Status: " + status);
+
+      Geolocation.setRNConfiguration({
+        skipPermissionRequests: false,
+        authorizationLevel: 'whenInUse',
+      });
+      Geolocation.requestAuthorization();
```

Comment:
Change: new method `requestAllPermissions` to ask multiple permissions at once, especially for Android 13 where we have to ask both location & notification permissions. Will ask notification permission first then location. If ask at the same time, only 1 will show.